File: ios/objNative/JSBufferEndType.m
//
//  JSBufferEndType.m
//  Supermap
//
//  Created by 王子豪 on 2017/8/16.
//  Copyright © 2017年 Facebook. All rights reserved.
//

#import "JSBufferEndType.h"
#import "SuperMap/BufferEndType.h"
@implementation JSBufferEndType
    RCT_EXPORT_MODULE();
-(NSDictionary*)constantsToExport{
    return @{@"ROUND":@(ROUND),
             @"FLAT":@(FLAT)};
}
@end

File: android/src/main/java/com/supermap/rnsupermap/JSBufferEndType.java
package com.supermap.rnsupermap;

import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;

import java.util.HashMap;
import java.util.Map;

/**
 * Created by Myself on 2017/8/16.
 */

public class JSBufferEndType extends ReactContextBaseJavaModule {
    public static final String REACT_CLASS="JSBufferEndType";

    public JSBufferEndType(ReactApplicationContext context){
        super(context);
    }

    @Override
    public String getName(){return REACT_CLASS;}

    @Override
    public Map<String, Object> getConstants() {
        final Map<String, Object> constants = new HashMap<>();
        constants.put("ROUND", 1);
        constants.put("FLAT", 2);
        return constants;
    }
}
